meta-lmp-bsp/recipes-security/optee/optee-os-fio-mfgtool_3.10.0.bbappend

```diff
@@ -33,6 +33,12 @@ EXTRA_OEMAKE_append_apalis-imx6 = " \
 EXTRA_OEMAKE_append_apalis-imx8 = " \
     CFG_UART_BASE=0x5a070000 \
 "
+EXTRA_OEMAKE_append_apalis-imx8-sec = " \
+    CFG_UART_BASE=0x5a070000 \
+    CFG_REE_FS=n CFG_RPMB_FS=y CFG_RPMB_FS_DEV_ID=0 \
+    CFG_EARLY_TA=y \
+    CFG_IN_TREE_EARLY_TAS=fiovb/22250a54-0bf1-48fe-8002-7b20f1c9c9b1 \
+"
```

Comment:
Please move this below the settings for secure machines: https://github.com/foundriesio/meta-lmp/blob/master/meta-lmp-bsp/recipes-security/optee/optee-os-fio-mfgtool_3.10.0.bbappend#L50